src/core/state.go

```diff
@@ -1353,18 +1353,47 @@ func (state *BuildState) IterInputs(target *BuildTarget, test bool) iter.Seq[Bui
 		return IterInputs(state, state.Graph, target, true, target.IsFilegroup)
 	}
 	return func(yield func(BuildInput) bool) {
+		// The target itself, plus its transitive run-time dependencies (since we're about to run the target):
 		if !yield(target.Label) {
 			return
 		}
+		for runDep := range target.IterAllRuntimeDependencies(state.Graph) {
+			log.Debug("state.IterInputs: %s", runDep.String())
```

Comment:
I think this log line either needs to be removed or have more context added to it

ditto below